tdei-ui/src/routes/Services/ClipBoardCopy.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from "react";
+import copyIcon from "../../assets/img/copy-icon.svg";
+import copiedIcon from "../../assets/img/clipboard-copied.svg"
+///
+/// Copy to ClipBoard Component
+///
+function ClipboardCopy({ copyText }) {
```

Comment:
I see this is new component, but please check we have already added plugin for copyclipboard and using in below file tdei-ui/src/components/UserHeader/UserHeader.js .

Reply:
Component is present but it is behaving differently. Tried integrating the same here also but facing css issues. Attaching screenshot
<img width="1470" alt="Screenshot 2023-09-11 at 4 18 21 PM" src="https://github.com/TaskarCenterAtUW/TDEI-user-management-front-end/assets/142565248/0daa5d56-8ba6-4b9e-a46f-64406656ac4a">

Reply:
Updated code with 'copyclipboard'. Please check the changes.